src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/domain/ÅrsakAvslag.kt

```diff
@@ -1,13 +1,17 @@
 package no.nav.tilleggsstonader.sak.vedtak.domain
 
-enum class ÅrsakAvslag {
-    INGEN_AKTIVITET,
-    IKKE_I_MÅLGRUPPE,
-    INGEN_OVERLAPP_AKTIVITET_MÅLGRUPPE,
-    MANGELFULL_DOKUMENTASJON,
-    HAR_IKKE_UTGIFTER,
-    RETT_TIL_UTSTYRSSTIPEND,
-    HAR_IKKE_MERUTGIFTER,
-    RETT_TIL_BOSTØTTE,
-    ANNET,
+enum class ÅrsakAvslag(
+    val displayName: String,
+) {
+    INGEN_AKTIVITET("Ingen aktivitet"),
+    IKKE_I_MÅLGRUPPE("Ikke i målgruppe"),
+    INGEN_OVERLAPP_AKTIVITET_MÅLGRUPPE("Ingen overlapp aktivitet målgruppe"),
+    MANGELFULL_DOKUMENTASJON("Mangelfull dokumentasjon"),
+    HAR_IKKE_UTGIFTER("Har ikke utgifter"),
+    RETT_TIL_UTSTYRSSTIPEND("Rett til utstyrsstipend"),
+    HAR_IKKE_MERUTGIFTER("Har ikke merutgifter"),
```

Comment:
Ser at vi flere steder har behov for en `displayName` på denne måten. 

I `BehandlingStatus` og `StegType` er det løst ved å lage en funksjon som utleder det:

```
    fun visningsnavn(): String =
        this.name
            .replace('_', ' ')
            .lowercase()
            .replaceFirstChar { it.uppercase() }
```

Hvis vi lager en util-funksjon kan det gjenbrukes også:

```
    fun visningsnavn(): String = enumNameToDisplayName(this.name)
```